homeassistant/components/rpi_gpio.py

```diff
@@ -48,6 +48,20 @@ def setup_input(port, pull_mode):
                GPIO.PUD_DOWN if pull_mode == 'DOWN' else GPIO.PUD_UP)
 
 
+def run_pwm(hass, port, frequency, duty_cycle, duration):
+    """Run GPIO as PWM."""
+    import RPi.GPIO as GPIO
+    from homeassistant.helpers.event import track_point_in_time
```

Comment:
Move this import to the top of the module. Only component specific libraries part of the requirements list need to be imported inside functions.